app/templates/partials/header.html

```diff
@@ -24,6 +24,13 @@
                 <div class="grid__col {{ 'col-10@xs col-8@s' if account_service_url }}">
                     <h1 class="header__title ">{{ survey_title }}</h1>
                 </div>
+                {% if account_service_log_out_url %}
+                    {% if not content or content.block and content.block.type == 'Introduction' %}
```

Comment:
I found this kinda confusing, can we use parentheses?.
`{% if not content or (content.block and content.block.type == 'Introduction') %}`